src/main/java/com/teragrep/lsh_01/Main.java

```diff
@@ -75,10 +78,14 @@ public static void main(String[] args) {
                         null,
                         200,
                         internalEndpointUrlConfig
-                )
+                ); Metrics metrics = new Metrics(metricsConfig.prometheusPort, metricRegistry)
         ) {
+            metrics.start();
             server.run();
         }
+        catch (Exception e) {
+            throw new RuntimeException("Couldn't close DropWizard metrics: " + e.getMessage());
```

Comment:
should this indicate could not start? What if server.run() throws as well then the error is misleading